app/views/catalog/_downloads_dropdown.html.erb

```diff
@@ -0,0 +1,27 @@
+<%# Renders the options of the downloads dropdown button %>
+<% document ||= @document %>
+
+<% if document.direct_download.present? %>
+  <% if document.direct_download[:download].is_a? Array %>
+    <% document.direct_download[:download].each do |download| %>
+      <%= download_link_file(download['label'], document.id, download['url']) %>
+    <% end %>
+  <% end %>
+  <% if document.direct_download[:download].is_a? String %>
+    <%= download_link_file(download_text(document.file_format), document.id, document.direct_download[:download]) %>
+  <% end %>
+<% end %>
+
+<% if document.hgl_download.present? %>
+  <%= download_link_hgl(download_text(document.download_types.first[0]), document) %>
+<% end %>
+
+<% if document.iiif_download.present? %>
+  <%= download_link_iiif %>
+<% end %>
+
+<% if document.download_types.present? %>
+  <% document.download_types.each do |type| %>
+    <%= download_link_generated(type.first, document) %>
```

Comment:
I think we want this to keep the translation behavior that was previously in the helper method `download_generated_body`